CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing
+
+Thanks for contributing! Our community of developers is what put SmartThings on the map!
+
+## How can I contribute?
+
+### Improve documentation
+
+As a user of the SmartThings CLI, you're the perfect candidate to help us improve our documentation. Error fixes, typo corrections, better explanations, more examples, etc. Open issues for things that could be improved – anything. Even improvements to this document.
+
+### Give feedback on issues
+
+We're always looking for more opinions on discussions in the issue tracker. It's a good opportunity to influence the future direction of the CLI.
+
+### [Submitting an issue or feature request](/issues/new/choose)
```

Comment:
These links are broken and navigating to them is pretty general Github knowledge, so I am going to just remove the hyperlinks.